tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/etc/ToEtcExpr.scala

```diff
@@ -627,23 +627,59 @@ class ToEtcExpr(annotationStore: AnnotationStore, varPool: TypeVarPool) extends
           accessorsWithNewValues.flatMap(p => List(this(p._1), this(p._2)))
         mkApp(ref, disjunctiveType, this(fun) +: xargs: _*)
 
-      case OperEx(TlaFunOper.recFunDef, body, NameEx(name), bindingSet) =>
-        // the expected type is: (((a -> b) => (a => b)) => (a -> b)) (λ $recFun ∈ Set(c -> d). λ x ∈ Int. x)
+      case OperEx(TlaFunOper.recFunDef, body, args @ _*) =>
+        // fun[x \in S, y \in T |-> e] == ...
+        // or, fun[<<x, y>> \in S, z \in T |-> e] == ...
+        //
+        // The expected type is for a one-argument function is:
+        // (((b -> a) => (b => a)) => (b -> a)) (λ $recFun ∈ Set(c -> d). λ x ∈ Int. x)
+        //
+        // (The case of multiple arguments is quite complex)
```

Comment:
Is this comment meant to say something about the code on line 638 or meant to explain why there's not a comment giving the type for a multiple argument function?

Reply:
It was referring to the case above, which is only single argument